### PR TITLE
Update management contacts

### PR DESCRIPTION
**Pull request by siinus**

### CODE REVIEW

File: assets/management-images/Erki_Lasberg.jpg
Comment (ultimate-evamarie) [inline assets/management-images/Erki_Lasberg.jpg:1]: Ei meeldi

**Review by ultimate-evamarie — CHANGES_REQUESTED** [review]
Pildid - ei meeldi

File: assets/management-images/Erki_Lasberg.jpg
Comment (ultimate-evamarie) [reply assets/management-images/Erki_Lasberg.jpg:1]: I want some changes NOW

Comment (ultimate-evamarie) [reply assets/management-images/Erki_Lasberg.jpg:1]: jk

**Review by ultimate-evamarie — APPROVED** [review]
lgtm

**Review by jeuzee — APPROVED** [review]
yes

**Pull request merged**